### PR TITLE
BaSP-MR-212: fixed messages modal

### PR DESCRIPTION
**Pull request by Eugeniagassman**
fixed message modal

### CODE REVIEW

**Comment by LucianoStradiot** [comment]
[[FIX] - Fix message when you insert a valid email on recovery password screen](https://trello.com/c/bILxcYYp/212-fix-fix-message-when-you-insert-a-valid-email-on-recovery-password-screen)

File: src/Views/login/recoverPassword.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const recoveryPassword = () => {
   const history = useHistory();
   const dispatch = useDispatch();
   const loading = useSelector((state) => state.recoverPassword.isLoading);
-  const error = useSelector((state) => state.recoverPassword.error);
+  // const error = useSelector((state) => state.recoverPassword.error);
```
Comment (Tilajc) [inline src/Views/login/recoverPassword.js]: delete this comment
```suggestion
 
```

**Review by Tilajc — APPROVED** [review]
Pretty good job! just delete this comment

**Comment by LautaroVercellini** [comment]
Works Good!

**Review by LautaroVercellini — APPROVED** [review]

**Pull request merged**